README.md

```diff
@@ -1,49 +1,78 @@
-# clear-jupyter
-Jupyter notebook Docker built on top of Clear Linux
-=======
+# spark-tk-jupyter
 
-- The final images contains pandas-cookbook example notebooks from this repository:
+Jupyter notebook docker built on top of Clear Linux*.
+
+>The final images contains pandas-cookbook example notebooks from this repo:
 https://github.com/jvns/pandas-cookbook
 
+## What's new
+
+This is the initial release of the `spark-tk-jupyter` repo.
+
+## Known issues
+
+None.
+
 ## Building the image
-- Pull all the submodules: git submodule update --init --recursive
-- sudo docker build --tag=spark-tk-jupyter .
-- Or if you are behind a proxy use this:
-- sudo docker build --build-arg HTTP_PROXY=$http_proxy --build-arg HTTPS_PROXY=$http_proxy --build-arg NO_PROXY=$no_proxy --build-arg http_proxy=$http_proxy --build-arg https_proxy=$http_proxy --build-arg no_proxy=$no_proxy --tag=spark-tk-jupyter .
+1. Pull all the submodules:  
+    `git submodule update --init --recursive`  
+2. Build the image:  
+    `sudo docker build --tag=spark-tk-jupyter`  
+
+   Or if you are behind a proxy use this:  
+    ```
+    sudo docker build --build-arg HTTP_PROXY=$http_proxy --build-arg HTTPS_PROXY=$http_proxy --build-arg NO_PROXY=$no_proxy --build-arg http_proxy=$http_proxy --build-arg https_proxy=$http_proxy --build-arg no_proxy=$no_proxy --tag=spark-tk-jupyter
+    ```  
 
-## Run the image:
-- sudo docker run -p 8900:8888 YOUR_JUPYTER_IMAGE_TAG
+## Run the image:  
+
+    sudo docker run -p 8900:8888 YOUR_JUPYTER_IMAGE_TAG
 
 ##Features
 
 - PySpark, Spark Shell
-- Jupyter REST API's for upload and running PySpark/SparkTK scripts
+- Jupyter REST APIs for upload and running PySpark/SparkTK scripts
 - ATK libraries
 - Spark-TK libraries
 - Python and Pip 2.7
-- Examples notebooks from the project jupyter-default-notebooks
+- Examples notebooks from the project `jupyter-default-notebooks`
```

Comment:
It is part of the repository. 
Only pandas-cookbook is an external sub-module and we want it to be like that because we don't own that repository.